changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 8.9.0 - xxxx-xx-xx =
+* Fix - Display a notice if taxes vary by customer's shipping address when checking out using the Stripe Express Checkout Element.
```

Comment:
Nit: Should it not be "...by customer's billing address..."

Reply:
Good catch! Updated here and the entry in readme.txt as well ✅